opentelemetry-proto/tests/grpc_build.rs

```diff
@@ -94,8 +94,10 @@ fn build_tonic() {
     }
 
     // add custom serializer and deserializer for AnyValue
-    builder = builder
-        .field_attribute("common.v1.KeyValue.value", "#[cfg_attr(feature =\"with-serde\", serde(serialize_with = \"crate::proto::serializers::serialize_to_value\", deserialize_with = \"crate::proto::serializers::deserialize_from_value\"))]");
+    for path in ["common.v1.KeyValue.value", "logs.v1.LogRecord.body"] {
+        builder = builder
+        .field_attribute(path, "#[cfg_attr(feature =\"with-serde\", serde(serialize_with = \"crate::proto::serializers::serialize_to_value\", deserialize_with = \"crate::proto::serializers::deserialize_from_value\"))]");
+    }
```

Comment:
LogRecord::body is `AnyValue`, and we need a custom serializer/deserializer for it.

Reply:
Yep that's a good follow up but not necessary blocking

Reply:
Thanks, this has been added to this PR, as was a pre-requisite to serialize/deserialize LogData.